src/OpenTK/Platform/MacOS/Cocoa/NSDragOperation.cs

```diff
@@ -0,0 +1,46 @@
+﻿#region License
+//
+// NSEventType.cs
+//
+// Author:
+//       Olle Håkansson <ollhak@gmail.com>
+//
+// Copyright (c) 2014 Olle Håkansson
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+#endregion
+
+using System;
+
+namespace OpenTK.Platform.MacOS
+{
+    enum NSDragOperation : int
```

Comment:
Sure, but can you throw a quick link in to where you got it, or write a brief line about what it does?
NSDragOperation isn't something immediately obvious to someone coming across the code. Ideally we don't want to require that people have to google stuff or know the API to be able to work with it.

Something like 'Used to determine the types of drag operations that may be performed by the user. See <some link> for info on what these mean."